shoop/admin/generated_resources.txt

```diff
@@ -0,0 +1,9 @@
+static/shoop_admin/js/picotable.js
```

Comment:
Would be nice if these files had a header informing the user that this is an autogenerated manifest...